smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolServiceClient.kt

```diff
@@ -147,7 +157,11 @@ open class HttpProtocolServiceClient(
 
     private fun renderEmptyAsynchronousConfigInitializer(properties: List<ConfigProperty>) {
         writer.openBlock("public convenience required init() async throws {", "}") {
```

Comment:
Below in this file, several initializers and calls to initializers are reformatted.  The `renderProperties()` function, defined at the bottom of this file, handles comma separation if desired.